spec/features/staff/users_can_create_a_budget_spec.rb

```diff
@@ -134,6 +134,7 @@
         project_activity = create(:project_activity,
           parent: programme_activity,
           organisation: user.organisation)
+        _report = create(:report, state: :active, organisation: user.organisation, fund: fund_activity)
```

Comment:
Agree that this is a bit lumpy but I'm not sure of a better way we can do it right now given the Report needs to belong to a specific fund/org. We can probably put this on the to refactor list ;)

Reply:
Yeah, I feel like we'll be setting the org and fund every time so having a factory won't help.

Also, that indentation!! I'll sort it!

Reply:
Ooopsie! Indentation was right the first time! 🤣